app/models/okta_redis/grants.rb

```diff
@@ -8,34 +8,51 @@ module OktaRedis
   class Grants < Model
     CLASS_NAME = 'GrantsService'
 
+    after_initialize do |grants|
+      grants.instance_variable_set(:@activity_log, [])
+    end
+
     def all
-      okta_response.body
+      all_grants = okta_response.body
+      log_message_to_sentry('Getting all grants', :info,
```

Comment:
`okta_response` will add some data too `@activity_log` and then we'll log all of that to sentry before returning the okta_response.body